public_wallet_app/css/styles.css

```diff
@@ -95,6 +95,15 @@ button.danger {
 button.danger:hover {
   background-color: #9c2929;
 }
+button.positive {
+  background-color: #38a14f;
```

Comment:
please don't define colors in place - ideally use one of those already in defined on top (look around styles.css on how it's done), and if you really need a new one add one there